### PR TITLE
Update 04-ggplot2.Rmd - use geom_count instead of geom_jitter

### PR DESCRIPTION
**Pull request by bkmgit**
Add additional plot that illustrates quantities better. If worth including, probably want to modify this part of the lesson a little to keep time. Maybe replace all geom_jitter plots with geom_point. Can reduce discussion on using alpha, and keep discussion on using color and add different shapes. Could also use [geom_count](https://ggplot2.tidyverse.org/reference/geom_count.html) instead of  geom_jitter.

### CODE REVIEW

**Comment by bkmgit** [comment]
Ready for review

**Comment by bkmgit** [comment]
Ok. Perhaps can add some explanation of the different legends.

**Comment by kelseygonzalez** [comment]
Thanks for taking the time to make this contribution, @bkmgit ! We always appreciate the new ideas the community proposes.

I actually find this change to `geom_count()` a bit harder to interpret than `geom_jitter()` because you can see multiple size points of different colors occupying the same space once `aes(color = village)` is introduced. This is really evident in the subsequent exercise. Moreover, geom_jitter is used again when learning boxplots in the next section and by replacing `geom_jitter()` with `geom_count()`, there's no explanation about jitter, what it does, or what it's used for. 

In short, I prefer multiple points (jitter) over differentiating sizes (count) because it seems like a more intuitive way to interpret density of points in a plot. @atheobold, what do you think? Definitely render the branch locally using `make serve` to check out how these changes look.

**Comment by bkmgit** [comment]
@kelseygonzalez @juanfung  Thanks. Removed jitter entirely. Added explanation for count and used it throughout. Might it be worth using alpha settings so that if there are multiple points of the same size at one point, this will be clearer?

**Comment by juanfung** [comment]
@atheobold These are all good points.

@bkmgit Would you be interested in instead adding an optional section on other aesthetics, to include `geom_count` and possibly other useful aesthetics beyond the introductory material? I think this would be a great enhancement to the lesson.

**Comment by bkmgit** [comment]
My worry is that the lesson is already long. Want to encourage some 
thought about other visualization options for that dataset, hence the 
suggestion to modify an exercise. The suggested paper is interesting, 
but ggplot may need some extension to make a very nice plot using the 
suggested glyphs.

It seems one can change the jitter options to control the amount of 
jitter, so that clustering occurs around the values, see for example:
https://thomasleeper.com/Rcourse/Tutorials/jitter.html
Might something like that be better?

**Comment by atheobold** [comment]
@bkmgit - I agree that adding details to the conversation around `geom_jitter()` could be better than adding new content. The lesson currently does not motivate the use of `geom_jitter()` outside of saying that we are trying to avoid overplotting. 

I think the following changes could make the discussion of `geom_jitter()` a bit more robust:
- describe how we can tell that there is overplotting going on
- itemize a set of methods to avoid overplotting (e.g. transparency, jittering) 
    * Walk through each change and describe what it does
 
The section on using the `alpha` argument motivates why it is useful, but **does not** describe *how* `alpha` is used. Such as, what numbers makes points lighter? What is the baseline `alpha`? 

The section on adding a jitter to the points needs substantially more information! We should introduce what jittering does, how we can use `geom_jitter()` to jitter points, and discuss how one can choose how much jittering is to be added. 

@benmarwick I am happy to help you make these changes, as I think they will substantially improve the lesson. Let me know what components you would prefer to tackle!

**Comment by bkmgit** [comment]
@atheobold <https://github.com/atheobold> @ajuanfung 
<https://github.com/juanfung> Thanks. Will make changes later this week.

**Comment by bkmgit** [comment]
@atheobold  Updated text. Feel free to make further changes.

**Comment by atheobold** [comment]
@bkmgit thanks for the updates! I added more language to motivate how we can tell there is an overplotting issue, and described two possible methods for alleviating overplotting. I moved your notes on the `geom_count()` function back to the scatterplot section, as I felt it flowed more naturally there.

**Review by kelseygonzalez — APPROVED** [review]
I made two commits to a) make 0.25 and 0.2 values in `geom_jitter()` consistent and b) fix a duplicate chunk error that was breaking the build of the site. Besides that, I am really happy with the changes from @bkmgit and @atheobold . They made the lesson clearer and more consistent.

**Review by juanfung — APPROVED** [review]
Awesome work @bkmgit and @atheobold !
